roles/lib_utils/filter_plugins/oo_filters.py

```diff
@@ -751,7 +723,6 @@ def filters(self):
             "lib_utils_oo_parse_named_certificates": lib_utils_oo_parse_named_certificates,
             "lib_utils_oo_parse_certificate_san": lib_utils_oo_parse_certificate_san,
             "lib_utils_oo_generate_secret": lib_utils_oo_generate_secret,
-            "lib_utils_oo_pods_match_component": lib_utils_oo_pods_match_component,
```

Comment:
Nitpick: `lib_utils_oo_chomp_commit_offset` is also not used anywhere

Reply:
I was thinking about checking them again.  I'll go ahead and remove these.